src/Microsoft.AspNet.Razor.Test.Sources/project.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "version": "1.0.0-*",
```

Comment:
Odd for this project to be inconsistent with the rest of the repo. Suggest `4.0.0-*`